src/app/components/mod.rs

```diff
@@ -63,7 +63,8 @@ pub use playlist_details::*;
 mod window;
 pub use window::*;
 
-mod utils;
+pub mod utils;
+pub use utils::*;
```

Comment:
since you're always using "use crate::app::components::**utils**::format_duration;" there is no need to reexport (pub use) the content of utils, ie you should be able to remove this line with no impact unless I am missing something?